JSDemos/Demos/TreeView/ItemSelectionAndCustomization/Vue/App.vue

```diff
@@ -35,37 +35,38 @@
     </div>
     <div class="options">
       <div class="caption">Options</div>
-      <div class="options-container">
-        <div class="option">
-          <span>Show Check Boxes Mode:</span>
-          <DxSelectBox
-            :items="showCheckBoxesModes"
-            v-model:value="showCheckBoxesModeValue"
-            @value-changed="showCheckBoxesModeValueChanged"
-          />
-        </div>
-        <div class="option">
-          <span>Selection Mode:</span>
-          <DxSelectBox
-            :items="selectionModes"
-            v-model:value="selectionModeValue"
-            :disabled="isSelectionModeDisabled"
-            @value-changed="selectionModeValueChanged"
-          />
-        </div>
-        <div class="option option-checkbox">
-          <DxCheckBox
-            text="Select Nodes Recursive"
-            :disabled="isRecursiveDisabled"
-            v-model:value="selectNodesRecursiveValue"
-          />
-        </div>
-        <div class="option option-checkbox">
-          <DxCheckBox
-            text="Select By Click"
-            v-model:value="selectByClickValue"
-          />
-        </div>
+      <div class="option">
+        <span>Show Check Boxes Mode:</span>
+        <DxSelectBox
+          :items="showCheckBoxesModes"
+          v-model:value="showCheckBoxesModeValue"
+          @value-changed="showCheckBoxesModeValueChanged"
+        />
+      </div>
+      {{ ' ' }}
+      <div class="option">
+        <span>Selection Mode:</span>
+        <DxSelectBox
+          :items="selectionModes"
+          v-model:value="selectionModeValue"
+          :disabled="isSelectionModeDisabled"
+          @value-changed="selectionModeValueChanged"
+        />
+      </div>
+      {{ ' ' }}
+      <div class="option recursive-option">
+        <DxCheckBox
+          text="Select Nodes Recursive"
+          :disabled="isRecursiveDisabled"
+          v-model:value="selectNodesRecursiveValue"
+        />
+      </div>
+      {{ ' ' }}
```

Comment:
Do you think this approach shows the best practices for Vue. And we recommend our customers to copy this code? Do not forget why these demos are.